docs/source/user/installing.rst

```diff
@@ -280,36 +281,41 @@ further information.
 
     __Hardware Information__
-    Machine                             : x86_64
-    CPU Name                            : haswell
-    CPU Features                        :
-    aes avx avx2 bmi bmi2 cmov cx16 f16c fma fsgsbase lzcnt mmx movbe pclmul popcnt
-    rdrnd sse sse2 sse3 sse4.1 sse4.2 ssse3 xsave xsaveopt
+    Machine                                       : x86_64
+    CPU Name                                      : skylake-avx512
+    CPU Count                                     : 12
+    CPU Features                                  :
+    64bit adx aes avx avx2 avx512bw avx512cd avx512dq avx512f avx512vl bmi bmi2
+    clflushopt clwb cmov cx16 cx8 f16c fma fsgsbase fxsr invpcid lzcnt mmx
+    movbe pclmul pku popcnt prfchw rdrnd rdseed rtm sahf sse sse2 sse3 sse4.1
+    sse4.2 ssse3 xsave xsavec xsaveopt xsaves
 
     __OS Information__
-    Platform                            : Darwin-17.6.0-x86_64-i386-64bit
-    Release                             : 17.6.0
-    System Name                         : Darwin
-    Version                             : Darwin Kernel Version 17.6.0: Tue May  8 15:22:16 PDT 2018; root:xnu-4570.61.1~1/RELEASE_X86_64
-    OS specific info                    : 10.13.5   x86_64
+    Platform Name                                 : Linux-5.4.0-94-generic-x86_64-with-glibc2.31
+    Platform Release                              : 5.4.0-94-generic
+    OS Name                                       : Linux
+    OS Version                                    : #106-Ubuntu SMP Thu Jan 6 23:58:14 UTC 2022
 
     __Python Information__
-    Python Compiler                     : GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)
-    Python Implementation               : CPython
-    Python Version                      : 2.7.15
-    Python Locale                       : en_US UTF-8
+    Python Compiler                               : GCC 9.4.0
+    Python Implementation                         : CPython
+    Python Version                                : 3.10.2
+    Python Locale                                 : en_GB.UTF-8
 
     __LLVM information__
-    LLVM version                        : 6.0.0
+    LLVM Version                                  : 11.1.0
 
     __CUDA Information__
     Found 1 CUDA devices
-    id 0         GeForce GT 750M                              [SUPPORTED]
-                          compute capability: 3.0
-                               pci device id: 0
-                                  pci bus id: 1
+    id 0      b'Quadro RTX 8000'                              [SUPPORTED]
+                          Compute Capability: 7.5
+                               PCI Device ID: 0
+                                  PCI Bus ID: 21
+                                        UUID: GPU-e6489c45-5b68-3b03-bab7-0e7c8e809643
+                                    Watchdog: Enabled
+                 FP32/FP64 Performance Ratio: 32
```

Comment:
It could also have been Windows, but the main aim I had was presenting something that consistently looked like plausible output from Numba, rather than something like a Macbook with an RTX 8000.

Reply:
I see, thanks for updating it!